front/migrations/20230920_agent_config_drop_sid_unique_constraints.sql

```diff
@@ -0,0 +1,17 @@
+alter table
```

Comment:
see [here](https://metabase.dust.tt/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJuYXRpdmUiLCJuYXRpdmUiOnsicXVlcnkiOiJTRUxFQ1QgY29uLipcbiAgICAgICBGUk9NIHBnX2NhdGFsb2cucGdfY29uc3RyYWludCBjb25cbiAgICAgICAgICAgIElOTkVSIEpPSU4gcGdfY2F0YWxvZy5wZ19jbGFzcyByZWxcbiAgICAgICAgICAgICAgICAgICAgICAgT04gcmVsLm9pZCA9IGNvbi5jb25yZWxpZFxuICAgICAgICAgICAgSU5ORVIgSk9JTiBwZ19jYXRhbG9nLnBnX25hbWVzcGFjZSBuc3BcbiAgICAgICAgICAgICAgICAgICAgICAgT04gbnNwLm9pZCA9IGNvbm5hbWVzcGFjZVxuICAgICAgIFdIRVJFIHJlbC5yZWxuYW1lID0gJ2FnZW50X2NvbmZpZ3VyYXRpb25zJzsiLCJ0ZW1wbGF0ZS10YWdzIjp7fX0sImRhdGFiYXNlIjo0fSwiZGlzcGxheSI6InRhYmxlIiwicGFyYW1ldGVycyI6W10sInZpc3VhbGl6YXRpb25fc2V0dGluZ3MiOnt9fQ==)

Reply:
Thank you 🙏🏻